x/launch/keeper/grpc_query_vested_account_test.go

```diff
@@ -0,0 +1,123 @@
+package keeper
+
+import (
+	"strconv"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/query"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/tendermint/spn/x/launch/types"
+)
+
+// Prevent strconv unused error
+var _ = strconv.IntSize
+
+func TestVestedAccountQuerySingle(t *testing.T) {
+	keeper, ctx := setupKeeper(t)
+	wctx := sdk.WrapSDKContext(ctx)
+	msgs := createNVestedAccount(keeper, ctx, 2)
+	for _, tc := range []struct {
+		desc     string
+		request  *types.QueryGetVestedAccountRequest
+		response *types.QueryGetVestedAccountResponse
+		err      error
+	}{
+		{
+			desc: "First",
+			request: &types.QueryGetVestedAccountRequest{
+				ChainID: msgs[0].ChainID,
+				Address: msgs[0].Address,
+			},
+			response: &types.QueryGetVestedAccountResponse{VestedAccount: &msgs[0]},
+		},
+		{
+			desc: "Second",
+			request: &types.QueryGetVestedAccountRequest{
+				ChainID: msgs[1].ChainID,
+				Address: msgs[1].Address,
+			},
+			response: &types.QueryGetVestedAccountResponse{VestedAccount: &msgs[1]},
+		},
+		{
+			desc: "KeyNotFound",
+			request: &types.QueryGetVestedAccountRequest{
+				ChainID: strconv.Itoa(100000),
+				Address: strconv.Itoa(100000),
+			},
+			err: status.Error(codes.InvalidArgument, "not found"),
+		},
+		{
+			desc: "InvalidRequest",
+			err:  status.Error(codes.InvalidArgument, "invalid request"),
+		},
+	} {
+		tc := tc
+		t.Run(tc.desc, func(t *testing.T) {
+			response, err := keeper.VestedAccount(wctx, tc.request)
+			if tc.err != nil {
+				require.ErrorIs(t, err, tc.err)
+			} else {
```

Comment:
```suggestion
			} else {
			    require.NoError(t, err)
```

Reply:
https://github.com/tendermint/starport/pull/1402

Also more generally, when you modify Starport generated code some changes are very specific to our implementation but if you find general improvement for the template or something to fix don't hesitate to create an issue on Staport repo or directly create the PR if it's very small

Reply:
I created an issue to list all of them: https://github.com/tendermint/starport/issues/1409